avro/src/main/scala/magnolify/avro/AvroType.scala

```diff
@@ -190,24 +190,30 @@ object AvroField {
 
   private def id[T](tpe: Schema.Type): AvroField[T] = aux[T, T, T](tpe)(identity)(identity)
 
-  implicit val afBoolean = id[Boolean](Schema.Type.BOOLEAN)
-  implicit val afInt = id[Int](Schema.Type.INT)
-  implicit val afLong = id[Long](Schema.Type.LONG)
-  implicit val afFloat = id[Float](Schema.Type.FLOAT)
-  implicit val afDouble = id[Double](Schema.Type.DOUBLE)
-  implicit val afString =
-    aux[String, CharSequence, String](Schema.Type.STRING)(_.toString)(identity)
-  implicit val afUnit =
-    aux2[Unit, JsonProperties.Null](Schema.Type.NULL)(_ => ())(_ => JsonProperties.NULL_VALUE)
```

Comment:
I don't know why we had this. The generated data looked incorrect and having a data model containing a `Unit` field looks wrong to me. I replaced with a `afNull` in case we want to get a schema with null type.